lib/growthbook/util.rb

```diff
@@ -1,8 +1,10 @@
 # frozen_string_literal: true
 
+require 'base64'
 require 'fnv'
 require 'bigdecimal'
 require 'bigdecimal/util'
+require 'openssl'
```

Comment:
Are openssl and base64 required here?  Or should these be moved to the decryption_util file?